packages/notices/src/store/selectors.js

```diff
@@ -14,6 +14,8 @@ import { DEFAULT_CONTEXT } from './constants';
  */
 const DEFAULT_NOTICES = [];
 
+/** @typedef {import('./actions').WPNoticeAction} WPNoticeAction */
```

Comment:
If it's something we want to have convention to locate at top of file, I might suggest it be placed before the constant on the previous line.

Reply:
Agreed, my intention was to have all of the type definitions just after import statements as you suggested 👍

Reply:
Fixed